lake-framework/README.md

```diff
@@ -125,7 +103,7 @@ $ AWS_DEFAULT_REGION=eu-central-1
 
 Add the following dependencies to your `Cargo.toml`
 
-```toml
+```text
```

Comment:
It's sad that it does not work with proper lang annotations :(

Reply:
It's actually working. I was struggling while fixing tests. They are not convenient to debug when all of them are failing. This particular place and some other is fixable. I've updated it. Thanks for you comment!